src/Admin/Controllers/OrganizationsController.cs

```diff
@@ -215,7 +215,10 @@ public async Task<IActionResult> Edit(Guid id, OrganizationEditModel model)
 
         try
         {
-            await _stripeSyncService.UpdateCustomerEmailAddress(organization.GatewayCustomerId, organization.BillingEmail);
+            if (!string.IsNullOrWhiteSpace(organization.GatewayCustomerId) && !string.IsNullOrWhiteSpace(organization.BillingEmail))
+            {
+                await _stripeSyncService.UpdateCustomerEmailAddress(organization.GatewayCustomerId, organization.BillingEmail);
+            }
         }
         catch (StripeException stripeException)
```

Comment:
Is it worth rethrowing so that the entire operation is aborted, just because we can't update their email in Stripe?

It might be more appropriate to prompt the user ("unable to update Stripe, do you want to continue?"), or provide an override switch or something. A question for Product.